usecases/config/config_handler.go

```diff
@@ -103,9 +103,11 @@ type RuntimeOverrides struct {
 
 // UsageConfig holds configuration for usage data collection and upload
 type UsageConfig struct {
```

Comment:
I'm thinking that we could also put those settings to `modulecomponents/usage` package, as it's something that is only applicable to usage modules, wdyt? (can be done in following PR's if decided to go with it)

Reply:
let me do it